apidoc/Modules/CloudPush/CloudPush.yml

```diff
@@ -231,12 +231,22 @@ events:
         Touching it launches the app, and fires this event.
         
         This is only applicable if you have set [showTrayNotification](Modules.CloudPush.showTrayNotification) to true.
+    properties:
+      - name: payload
+        summary: A JSON string of your data payload. Use JSON.parse to turn this in to an object you can use.
+        type: String
```

Comment:
@hansemannn
No. The reference in documentation is missing. The behaviour of API is same.